docs/content/development/extensions-core/druid-basic-security.md

```diff
@@ -310,6 +310,18 @@ For information on what HTTP methods are supported on a particular request endpo
 
 GET requires READ permission, while POST and DELETE require WRITE permission.
 
+### SQL Permissions
+
+Queries on Druid datasources require DATASOURCE READ permissions for the specified datasource.
+
+Queries on the [information schema tables](../../querying/sql.html#information-schema) require DATASOURCE READ access for the specified datasource.
```

Comment:
Please adjust the language a bit- there is not necessarily any 'specified datasource' with the information schema. The query might just be `SELECT * FROM INFORMATION_SCHEMA.COLUMNS`. I'd suggest:

```
Queries on the [INFORMATION_SCHEMA tables](../../querying/sql.html#information-schema) will
return information about datasources that the caller has DATASOURCE READ access to. Other
datasources will be omitted.
```

Reply:
Thanks, reworded